components/post_view/post_attachment_opengraph/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getOpenGraphMetadataForUrl} from 'mattermost-redux/selectors/entities/posts';
+import {getBool} from 'mattermost-redux/selectors/entities/preferences';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+import {GenericAction} from 'mattermost-redux/types/actions';
+import {OpenGraphMetadata} from 'mattermost-redux/types/posts';
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {editPost} from 'actions/views/posts';
+import {GlobalState} from 'types/store';
+import {Preferences} from 'utils/constants';
+
+import PostAttachmentOpenGraph, {Props} from './post_attachment_opengraph';
+
+function mapStateToProps(state: GlobalState, ownProps: Props) {
+    const config = getConfig(state);
+
+    return {
+        currentUserId: getCurrentUserId(state),
+        enableLinkPreviews: config.EnableLinkPreviews === 'true',
+        openGraphData: getOpenGraphMetadataForUrl(
+            state,
+            ownProps.postId,
+            ownProps.link,
+        ) as OpenGraphMetadata,
```

Comment:
I assume this can be changed once mattermost-redux is updated?

Reply:
If I add every missing required property yes !